docs/cloud/azureai/use-flow-in-azure-ml-pipeline.md

```diff
@@ -133,8 +196,50 @@ Above is part of the pipeline job python code, see here for [full example](https
+
+```yaml
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Run.schema.json
+flow: <your-flow-directory>
+azureml:
+  environment: azureml:my-environment:1
+```
+
+For more details about the supported format of Azure ML environment, please refer to [this doc](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?view=azureml-api-2&tabs=cli).
+
 ## Difference across flow in prompt flow and pipeline job
 
 In prompt flow, flow runs on [runtime](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/concept-runtime), which is designed for prompt flow and easy to customize; while in pipeline job, flow runs on different types of compute, and usually compute cluster.
 
 Given above, if your flow has logic relying on identity or environment variable, please be aware of this difference as you might run into some unexpected error(s) when the flow runs in pipeline job, and you might need some extra configurations to make it work.
+
+## Component ports and run settings
+
+|                  | port name                                           | source                                           | type                    | description                                                  |
+| ---------------- | --------------------------------------------------- | ------------------------------------------------ | ----------------------- | ------------------------------------------------------------ |
+| input ports      | data                                                | fixed                                            | uri_folder              | required; to pass in input data. Supported format includes [`mltable`](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-mltable?view=azureml-api-2&tabs=cli#authoring-mltable-files) and list of jsonl files. |
+|                  | run_outputs                                         | fixed                                            | uri_folder              | optional; to pass in output of a standard flow for [an evaluation flow](../../how-to-guides/develop-a-flow/develop-evaluation-flow.md). Should be linked to a `flow_outputs` of a previous flow node in the pipeline. |
+|                  | \<flow-input-name\>                                 | from flow inputs                                 | string                  | default value will be inherited from flow inputs; used to override [column mapping](../../how-to-guides/run-and-evaluate-a-flow/use-column-mapping.md) for flow inputs. |
+| input parameters | connections.\<node-name\>.connection                | from nodes of built-in or custom LLM tools       | string                  | default value will be the current value defined in `flow.dag.yaml` or `run.yaml`; to override used connections of corresponding nodes. Connection should exist in current workspace. |
```

Comment:
maybe just nodes using connections?

Is LLM tools a must here?

Reply:
hmm... seems that LLM tools is a must here as parameters are generated based on type of node instead of type of node inputs, given LLM tool has no explicit connection inputs